backend/mobile-backend/schema.graphql

```diff
@@ -0,0 +1,60 @@
+# This "input" configures a global authorization rule to enable public access to
+# all models in this schema. Learn more about authorization rules here: https://docs.amplify.aws/cli/graphql/authorization-rules
+input AMPLIFY { globalAuthRule: AuthRule = { allow: public } } # FOR TESTING ONLY!
+
+type GameSession @model {
```

Comment:
I think we can add `updatedAt` and then set the same field in `timestamp` annotation. That way we can have it to be as the sort key and also have amplify handles it for us

Reply:
I added the updatedAt with timestamp annotation (pushed a new version, not sure if that is what you meant here) but Amplify doesn't like Sortkeys other than type of string

Reply:
Huh bummer, lets then see if we can have the state / isPhaseOne field to be the sortkey. If not, We can have that one also to be indexed

Reply:
After trying this, got the same complaint about type not being a String. Not quite sure what you mean there by index it

Reply:
Sounds good. We can look into it later to see what'd be alternatives